app.py

```diff
@@ -141,6 +129,69 @@ async def create_directory():
         return jsonify(["CreateDirectoryRequestError: " + str(error)]), 400
 
 
+@app.post("/image-validation")
+async def image_validation():
+    """
+    Validates an image based on its extension, header, size, and resizability.
+
+    Returns:
+        A JSON response containing a boolean indicating whether the image is valid and a validator hash.
+
+    Raises:
+        ImageValidationError: If the image fails any of the validation checks.
+    """
+    try:
+        valide_extension = {"jpeg", "jpg", "png", "gif", "bmp", "tiff", "webp"}
+        valide_dimension = [1920, 1080]
```

Comment:
Extract the constants from the try
```suggestion
    VALIDE_EXTENSION = {"jpeg", "jpg", "png", "gif", "bmp", "tiff", "webp"}
    VALIDE_DIMENSION = [1920, 1080]
    try:
```

Reply:
It would be VALID no E